packages/liveblocks-client/src/OthersRef.ts

```diff
@@ -110,12 +111,14 @@ export class OthersRef<
   setConnection(
     connectionId: number,
     metaUserId: TUserMeta["id"],
-    metaUserInfo: TUserMeta["info"]
+    metaUserInfo: TUserMeta["info"],
+    metaIsReadonly: User<TPresence, TUserMeta>["isReadonly"]
```

Comment:
```suggestion
    metaIsReadonly: boolean
```

Since this isnt a complicated type, for readability it may help to see that it’s just a boolean.